components/list/demo/list-item-sample.js

```diff
@@ -19,9 +20,28 @@ class ListItemSample extends ListItemCheckboxMixin(LitElement) {
 			[slot="control"] {
 				width: 40px;
 			}
+			.d2l-list-item-link {
+				display:block;
+				height:100%;
+			}
+			:host([href]) {
+				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
+			}
+			a[href]:focus + .d2l-list-item-content,
+			a[href]:hover + .d2l-list-item-content {
+				--d2l-list-item-content-text-decoration: underline;
+			}
```

Comment:
These will not trigger correctly on Legacy Edge and Firefox for some reason. Any input as to why or other ways to do this are welcome.

Reply:
One solution `:host(:hover) a + .d2l-list-item-content` not sure how I feel about it

Reply:
Won't that trigger even if something like the `outside-control-action` is being hovered?

Reply:
Yeah :( I knew there was a reason this was bad.

Reply:
I have fixed this using javascript event listeners. Not exactly the cleanest way to add styles, but it works.